torch_audiomentations/utils/file.py

```diff
@@ -0,0 +1,21 @@
+import os
+import glob
+import librosa
+
+
+SUPPORTED_EXTENSIONS = ['.wav']
+
+
+def find_files(path):
+    """Finds all files of supported extensions."""
+    files = []
+
+    for supported_extension in SUPPORTED_EXTENSIONS:
+        files.extend(glob.glob(os.path.join(path, '*' + supported_extension)))
+
+    return files
+
+def load_audio(audio_file_path, sample_rate):
+    """Loads the audio given the path of an audio file."""
+    return librosa.load(audio_file_path, sr=sample_rate)[0]
```

Comment:
I wouldn't force the sampling rate during loading. 
We can use `soundfile` or `torchaudio` (that we will need eventually) for loading, rather than librosa.

Reply:
@mpariente Would you instead want the code to raise a warning if the sample rate of the loaded file doesn't match the expected sample rate?

Reply:
Alternatively, raise a warning (about how resampling hurts performance) and then resample? I think I would prefer that

Reply:
Yes, that does make sense. But then, should we load and resample with torchaudio?

Reply:
Sure, I'm okay with that. It can also be the scope of a different pull request.

Reply:
https://github.com/asteroid-team/torch-audiomentations/issues/9